plugins/cilium-docker/driver/driver.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/cilium/cilium/common/addressing"
 	"github.com/cilium/cilium/common/plugins"
 	"github.com/cilium/cilium/pkg/client"
-	endpointPkg "github.com/cilium/cilium/pkg/endpoint"
+	endpointIDPkg "github.com/cilium/cilium/pkg/endpoint/id"
```

Comment:
Merge this commit with the id package commit in order not to break git bisecting as the previous commit alone will not compile.